pkg/controller/registry/reconciler/configmap.go

```diff
@@ -101,7 +101,7 @@ func (s *configMapCatalogSourceDecorator) Service() *v1.Service {
 }
 
 func (s *configMapCatalogSourceDecorator) Pod(image string) *v1.Pod {
-	pod := Pod(s.CatalogSource, "configmap-registry-server", image, "", s.Labels(), s.Annotations(), 5, 2)
+	pod := Pod(s.CatalogSource, "configmap-registry-server", image, "", s.Labels(), s.Annotations(), 5, 5)
```

Comment:
Minor nit: This seems like a magic number for people that are new to the codebase - I wonder if it's worth making this a constant in the reconcile package so we don't hardcode that value in two places.

Reply:
I don't mind creating two constants for `readinessDelay` and `livenessDelay`, but for 

> magic number for people that are new to the codebase

if you look at the function signature
```
Pod(source *v1alpha1.CatalogSource, name string, image string, saName string, labels map[string]string, annotations map[string]string, readinessDelay int32, livenessDelay int32)
```

it looks pretty self-explanatory. 

Creating two constants `LivenessDelay` and `ReadinessDelay` seems superfluous when we're only creating this pod in one place.

Reply:
That's fine with me - I wasn't recommending you change it, but throwing out the suggestion as I think it improves readability.

That function signature is brutal though, so moving a couple of parameters from hardcoded numbers to constants probably doesn't move the needle all that much. Refactoring that function to use a type, or reducing the number of parameters, doesn't seem worth the effort for a function with one call site.